CoreScriptsRoot/Modules/Settings/SettingsHub.lua

```diff
@@ -209,6 +220,91 @@ local function CreateSettingsHub()
     this.BottomBarButtons[#this.BottomBarButtons + 1] = {buttonName, hotKeyTable}
   end
 
+  local function shouldShowBumperHints(lastInputType)
+      return utility:IsGamepadInputType(lastInputType) and this.BumperHintBar.Size.Y.Offset > MINIMUM_BUMPER_HINT_HEIGHT       
+  end
+
+  local function createBumperHints()
+      this.BumperHintBar = utility:Create("Frame") {
+          Parent = this.Shield,
```

Comment:
Do we have to parent this to the shield, is there no lower level container that would be better to adorn our bumper hints to?